src/common/cockpittest.c

```diff
@@ -217,6 +217,7 @@ cockpit_test_init (int *argc,
   g_setenv ("GIO_USE_VFS", "local", TRUE);
   g_setenv ("GSETTINGS_BACKEND", "memory", TRUE);
   g_setenv ("GIO_USE_PROXY_RESOLVER", "dummy", TRUE);
+  g_setenv ("LANG", "C", TRUE);
```

Comment:
LC_ALL for robustness, in case the environment has other LC_* bits set.

Reply:
set